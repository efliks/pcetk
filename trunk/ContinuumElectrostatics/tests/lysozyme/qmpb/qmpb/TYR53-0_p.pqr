../0/site_TYR_53_p.pqr