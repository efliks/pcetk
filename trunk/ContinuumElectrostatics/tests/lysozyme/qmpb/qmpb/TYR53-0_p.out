Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named TYR53-0_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.919373, 20.3963, 19.0418)) 121 2
Finer level:    (9.756, 12.524, 22.362) 101 1
Finer level:    (9.756, 12.524, 22.362) 101 0.5
Finest level:   (9.756, 12.524, 22.362) 101 0.25


Self energy of TYR53-0_p in three dielectric environment = 126.813
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of TYR53-0_p with background charge set in three dielectric environment = 1.30057
(probably in kcal/mole)



Interaction energies of TYR53-0_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.00098713
0   1 : -1.79976e-05
1   0 : 7.64036e-05
1   1 : 0.000742199
2   0 : 5.32092e-05
2   1 : 3.13425e-06
3   0 : 4.94721e-05
3   1 : 6.41177e-05
3   2 : 2.33227e-05
3   3 : 0.000122675
4   0 : 4.89148e-05
4   1 : -0.000190349
5   0 : 0.000214874
5   1 : -0.0015372
6   0 : -0.000157494
6   1 : -0.00148777
7   0 : -0.000593083
7   1 : 1.76481e-06
8   0 : 0.000147474
8   1 : -0.00998086
9   0 : -0.00465321
9   1 : -0.0510014
10   0 : -0.00474423
10   1 : -0.116958
11   0 : 253.626
11   1 : 143.612
12   0 : -0.515852
12   1 : -3.45993
13   0 : -0.000166012
13   1 : 0.000809682
14   0 : 0.00112309
14   1 : 3.82237e-05
15   0 : 0.00312513
15   1 : -7.80937e-06
16   0 : -0.0001362
16   1 : -0.0073485
17   0 : 0.00116396
17   1 : -1.20268e-05
18   0 : 3.76011e-06
18   1 : -6.71233e-05
Total runtime: 7.75 s user 0.06 s system 
